database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -100,7 +101,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
         pipelineConfigurer, dbSinkConfig, ConnectionConfig.JDBC_PLUGIN_TYPE);
       if (driverClass != null && !dbSinkConfig.connectionParamsContainsMacro()) {
         FailureCollector collector = configurer.getFailureCollector();
-        validateSchema(collector, driverClass, dbSinkConfig.tableName, inputSchema);
+        validateSchema(collector, driverClass, dbSinkConfig.tableName, inputSchema, dbSinkConfig.dbSchemaName);
```

Comment:
Is there any specific reason to pass schema as a param to the method? `validateSchema` can always use `dbSinkConfig.getDBSchemaName()`. Looks like the existing code has the same problem.

Reply:
The newer dev branch has this change. we use the geTableName and getSchemaName methods. Make the change here?

Reply:
yeah, let's only use getter to access the configs

Reply:
ok. Modified the code to use methods